grunt.js

```diff
@@ -314,6 +300,23 @@ module.exports = function(grunt) {
         src: ['build/temp/video.js', 'node_modules/vtt.js/dist/vtt.js'],
         dest: 'build/temp/video.js',
       },
+    },
+    concurrent: {
+      options: {
+        logConcurrentOutput: true
+      },
+      // Run multiple watch tasks in parallel
+      // Needed so watchify can cache intelligently
+      watchAll: [
+        'watch',
+        'browserify:watch',
```

Comment:
we should move `browserify:watch` and `browser:tests` to the first two items in this list.
Seems like as it currently is, `browserify:tests` ends  up running after `karma:watch` runs so, it takes some time extra before you can refresh your browser window.